src/main/resources/view/DarkTheme.css

```
@@ -357,10 +473,16 @@
 }
 
 #tags .label {
-    -fx-text-fill: white;
-    -fx-background-color: #3e7b91;
+    -fx-cursor: hand;
+    -fx-text-fill: derive(-fx-base1, -30%);
```

Maybe change the label colour to white?
```suggestion
    -fx-text-fill: white;
```

---

Seems like a good idea, have make the suggested change